var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -680,49 +683,43 @@ def install(self, spec, prefix):
             env["PATH"] = newdir + ":" + env["PATH"]
 
         with_libs = list()
```

Comment:
Maybe smth like

```python
with_libs = {f"+{lib}" for lib in Boost.all_libs}

if ...:
    with_libs.discard(...)
```

instead of `remove_if_in_list`?

Only thing is that a set is unordered, so maybe use `sorted(with_libs)` later, or something.